server/handler/assignments.go

```diff
@@ -0,0 +1,71 @@
+package handler
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/src-d/code-annotation/server/repository"
+	"github.com/src-d/code-annotation/server/serializer"
+
+	"github.com/go-chi/chi"
+)
+
+func GetAssignmentsForUserExperiment() RequestProcessFunc {
+	return func(r *http.Request) (resp *serializer.Response, errs []*serializer.Error) {
+		experimentID, err := strconv.Atoi(chi.URLParam(r, "experimentId"))
+		if err != nil {
+			errs = append(errs, &serializer.Error{500, "Wrong format in experiment ID sent", err.Error()})
```

Comment:
sorry I don't remember in which document it's described. But `go vet` won't allow this code with warning like `composite literal uses unkeyed fields` https://golang.org/cmd/vet/#hdr-Unkeyed_composite_literals

And it makes a lot of sense!
If you change the order of fields (last 2) in your struct - code still compiles but works incorrectly.
Also it will be very painful to add a new optional field.